core/domain/stats_domain.py

```diff
@@ -876,6 +876,17 @@ def __init__(
         self.schema_version = schema_version
         self.is_valid = is_valid
 
+    def __eq__(self, other):
+        if isinstance(other, ExplorationIssue):
+            return (
+                self.issue_type == other.issue_type and
+                self.issue_customization_args == other.issue_customization_args
+                and self.playthrough_ids == other.playthrough_ids
+                and self.schema_version == other.schema_version
+                and self.is_valid == other.is_valid
+            )
+        return False
```

Comment:
You might need to introduce `__hash__` too.

Reply:
Implicit calling of ``` __hash__ ```while using ```object1 == object2``` is not done. (where object1 and object2 are the objects of same class) Hence in this class ``` __eq__()``` is only used.